Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1646,8 +1647,12 @@ void CGContextFillRects(CGContextRef context, const CGRect* rects, size_t count)
 */
 void CGContextDrawPath(CGContextRef context, CGPathDrawingMode mode) {
     NOISY_RETURN_IF_NULL(context);
-    UNIMPLEMENTED();
-    context->ClearPath();
+    if (context->HasPath()) {
+        ID2D1Geometry* pGeometry;
```

Comment:
Yes, yes, yes.

using raw * means you *really* have to think about it and its easy to forget something / mess up. Com stuff gets a ComPtr almost always.

---
In reply to: [86611265](https://github.com/Microsoft/WinObjC/pull/1307#discussion_r86611265) [](ancestors = 86611265)